dev/release.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+import click
+from datetime import datetime
+from six.moves import input
+from subprocess import check_call, check_output
+import sys
+
+DATABRICKS_REMOTE = "git@github.com:graphframes/graphframes.git"
+PUBLISH_MODES = {
+    "local": "publishLocal",
+    "m2": "publishM2",
+    "spark-package-publish": "spPublish",
+}
+PUBLISH_DOCS_DEFAULT = True
+
+WORKING_BRANCH = "WORKING_BRANCH_RELEASE_%s_@%s"
+# lower case "z" puts the branch at the end of the github UI.
+WORKING_DOCS_BRANCH = "zWORKING_BRANCH_DOCS_%s_@%s"
+RELEASE_TAG = "v%s"
+
+
+def prominentPrint(x):
+    x = str(x)
+    n = len(x)
+    lines = ["", "=" * n, x, "=" * n, ""]
+    print("\n".join(lines))
+
+
+def verify(prompt, interactive):
```

Comment:
We could use `click.confirm`
https://click.palletsprojects.com/en/7.x/prompts/#confirmation-prompts